### PR TITLE
Adds new items to the vending machine found on the derelict.

### PR DESCRIPTION
**Pull request by FudgePucker**
Removes the Chief Engineer jumpsuits from the vending machine found in derelict research.
Also adds some new jumpsuits and hard hats. (Fun Fact, there was 4 fire suits but no hardhats that had fire resist on them so the fire suits were pointless.)

Also this is my first Pull so I hope I don't fuck anything up too hard.

### CODE REVIEW

**Comment by DeityLink** [comment]
No f-ing way mate!

**Comment by ComicIronic** [comment]
Looks alright for a first PR.

**Comment by FudgePucker** [comment]
look at my second commit, bring a coin with you.
Also why won't it merge?
The fuck is the "No new line at end of file" thingy? Is its absence whats causing the compile error?

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1238,12 +1238,14 @@
 	icon_state = "engi"
 	icon_deny = "engi-deny"
 	req_access_txt = "11"
-	products = list(/obj/item/clothing/under/rank/chief_engineer = 4,/obj/item/clothing/under/rank/engineer = 4,/obj/item/clothing/shoes/orange = 4,/obj/item/clothing/head/hardhat = 4,
+	products = list(/obj/item/clothing/under/rank/engineer = 4,/obj/item/clothing/under/rank/atmospheric_technician = 4,/obj/item/clothing/under/rank/maintenance_tech/ = 4,/obj/item/clothing/under/rank/engine_tech = 4,/obj/item/clothing/under/rank/electrician = 4,
+					obj/item/clothing/shoes/orange = 4,/obj/item/clothing/head/hardhat = 4,/obj/item/clothing/head/hardhat/orange = 4,/obj/item/clothing/head/hardhat/red = 4,/obj/item/clothing/head/hardhat/white = 4,/obj/item/clothing/head/hardhat/dblue = 4,
```
Comment (ComicIronic) [inline code/game/machinery/vending.dm]: You are missing a / here

**Comment by d3athrow** [comment]
To figure out what exactly occurred to break the travis CI build, click on the details link details link or click on the red x next to a commit red x then click on one of the build links at the bottom and you can see the output. in your case its

```
loading baystation12.mdme
code/setup.dm:6:warning: #warn If you have issues with retrieving logs update byond on the server and client to 507.1277 or greater, or uncomment RUNWARNING
loading interface/skin.dmf
code/game/machinery/vending.dm:1242:error: obj/item/clothing/shoes/orange: undefined var
code/game/objects/effects/beam.dm:124:warning: master_ref: variable defined but not used
baystation12.mdme.dmb - 1 error, 2 warnings
```

**Comment by DeityLink** [comment]
Oh, so it's a premium item now. Ok I can get behind that.

**Comment by FudgePucker** [comment]
Alright.
Thanks Comic and Pomf.
What was the blue hardhat originally for anyways? I haven't seen one on station.

**Comment by DeityLink** [comment]
some of those clothes are older versions of the clothes currently on the station. Kept in there for nostalgia or whatever.

**Comment by FudgePucker** [comment]
Ah, well I loaded the test serb and spawned me one, it isn't some invisible object that was removed and didn't have any sprite errors so I went ahead and added it.

**Comment by FudgePucker** [comment]
I mean all this changes on a meta level is that you need to bring a coin with you, and MoMMIs get more hats to wear.

**Comment by FudgePucker** [comment]
Considering that it would be rude to make a new pull request changing the vending machine for a while, what contraband items would be funny to add to it?
I was thinking 2 foam extinguishers, a flare gun, some boxes of flares, and a pack of smokes and a lighter.

**Comment by DeityLink** [comment]
>Considering that it would be rude to make a new pull request changing the vending machine for a while

why?

**Comment by FudgePucker** [comment]
I don't know, them having to proof read and merge it twice?

**Comment by DeityLink** [comment]
"They"'ll do it anyway, any time you push a new commit to this PR.

**Comment by AutismSpider** [comment]
For  contraband items

two foam extinguishers
one or two socket wrenches
one or two Device Analyzers

**Comment by FudgePucker** [comment]
I like how this PR is titled removes CE jumpsuit, when its more like adds new items, contraband, and premium items to the engineering vending machine that is located on derelict.
Oh well.

**Comment by clusterfack** [comment]
You know you can retitle it right?

**Comment by clusterfack** [comment]
Well you do now apparently

**Comment by Kurfursten** [comment]
Welcome to the wonderful world of forgetting to make a changelog. Don't worry -- it's practically my trademark.

**Comment by DeityLink** [comment]
just make a PR with the changelogs you've forgotten.

**Pull request merged**